### ISSUE TITLE
Support multiple AsyncAPI files in AsyncAPI Generator Plugin

### ISSUE THREAD

**Issue by boyney123** [boyney123/eventcatalog#146]
### Have you read the Contributing Guidelines on issues?

- [X] I have read the [Contributing Guidelines on issues](https://github.com/boyney123/eventcatalog/blob/master/CONTRIBUTING.md#reporting-new-issues).

### Description

The [genertor-asyncapi plugin](https://www.eventcatalog.dev/docs/api/plugins/@eventcatalog/plugin-doc-generator-asyncapi) currently only supports one AsyncAPI file.

It would be nice to support multiple files.

Here is an example of how it could work

```
const path = require('path');

module.exports = {
   generators: [
    [
      '@eventcatalog/plugin-doc-generator-asyncapi',
      {
        // pass one
        pathToSpec: path.join(__dirname, 'asyncapi.yml'),

		// or pass more....
	    pathToSpec: [path.join(__dirname, 'asyncapi.yml'), path.join(__dirname, 'asyncapi2.yml')],

        // version events if already in catalog (optional)
        versionEvents: true
      },
    ],
  ],
};
```

### Motivation

Allow users to use more than one AsyncAPI file with generator (based on feedback from community on Discord)

**Comment by boyney123** [comment]
This might also be able to help too. https://github.com/asyncapi/bundler

We could take in all the AsyncAPI files and bundle them before we process them...

**Comment by boyney123** [comment]
Was fixed and implemented in #149
